ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -55,41 +63,56 @@ const getpaymentMethods = (
   {
     name: I18n.t("wallet.methods.card.name"),
     description: I18n.t("wallet.methods.card.description"),
+    icon: CreditCard,
     onPress: props.navigateToAddCreditCard,
     status: "implemented",
     section: "credit_card"
   },
   {
-    name: I18n.t("wallet.methods.postepay.name"),
-    description: I18n.t("wallet.methods.postepay.description"),
-    onPress: props.navigateToAddCreditCard,
-    status: "implemented",
-    section: "credit_card"
+    name: I18n.t("wallet.methods.paypal.name"),
+    description: I18n.t("wallet.methods.paypal.description"),
+    icon: PaypalLogo,
+    // TODO replace with effective dispatch
+    onPress: constNull,
+    status: payPalEnabled ? "implemented" : "notImplemented",
+    section: "digital_payments"
+  },
+  {
+    name: I18n.t("wallet.methods.bancomatPay.name"),
+    description: I18n.t("wallet.methods.bancomatPay.description"),
+    icon: BpayLogo,
+    status: !onlyPaymentMethodCanPay ? "implemented" : "notImplemented",
+    onPress: props.startBPayOnboarding,
+    section: "digital_payments"
+  },
```

Comment:
We need to change also the workunit for the BANCOMATPay onboarding, because now the starting screen of the workflow changed and after an addition has been completed we return in the wrong screen:
https://user-images.githubusercontent.com/26501317/140058388-672dec62-1a6d-4cac-bc06-3d481ef84126.mov

If this pr is doesn't have high priority, I suggest to wait the merge of https://github.com/pagopa/io-app/pull/3448 in order to fix the new navigation flow.

Reply:
I guess @Undermaken can answer about the prioritzation of both, anyway I guess we can wait and update the workunit once everything is update since this refactoring is included as a part o Paypal integration process

Reply:
Yes it can wait.
It should be available when PayPal is supported.
I can't see any problems to consider this PR after #3448 is merged